sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationAsyncClientTest.java

```diff
@@ -1216,6 +1216,7 @@ public void listRevisionsWithPaginationAndRepeatIterator(HttpClient httpClient,
      * Verifies that, given a ton of existing settings, we can list the ConfigurationSettings using pagination
      * (ie. where 'nextLink' has a URL pointing to the next page of results.
      */
+    @Disabled("Each page can contains 100 settings, but we are experience 403 TOO_MANY_REQUESTS error for size of 50.")
```

Comment:
nit: should be `Issue #1234 - 403 TOO_MANY_REQUESTS for >=50 settings (should support 100)`